src/resources/assets/js/ptpContainer.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+    <div class="ptp-container">
+        <form class="form-stacked">
+            <div class="form-group">
+                <h4>Create a new Point to Polygon job <small><span class="label label-warning">experimental</span></small></h4>
+                <span>Run the point to polygon transformation using Magic SAM</span><br>
+            </div>
+            <div class="form-group">
+              <button
+                  class="btn btn-success btn-block"
+                  type="button"
+                  title="Run Point to Polygon conversion on this volume"
```

Comment:
If the button is disabled because a job is already running, the title should change to something that explains this.